shims/spark330/src/main/scala/com/nvidia/spark/rapids/shims/spark330/Spark330Shims.scala

```diff
@@ -16,11 +16,18 @@
 
 package com.nvidia.spark.rapids.shims.spark330
 
+import ai.rapids.cudf.ColumnVector
 import com.nvidia.spark.rapids._
 import com.nvidia.spark.rapids.shims.v2._
 
 class Spark330Shims extends Spark33XShims {
   override def getSparkShimVersion: ShimVersion = SparkShimServiceProvider.VERSION
 
   override def isCastingStringToNegDecimalScaleSupported: Boolean = true
+
+  override def throwIndexOutOfBoundsException(
+      ordinalValue: Int,
+      minNumElements: Int): ColumnVector = {
+    org.apache.spark.RapidsSparkIndexOutOfBoundsException.throww(ordinalValue, minNumElements)
```

Comment:
Looks like a typo? `throww` should be `throwSparkArrayIndexOutOfBoundsException`?